Mlem/Views/Tabs/Settings/Components/Settings View.swift

```diff
@@ -21,8 +25,57 @@ struct SettingsView: View {
             ScrollViewReader { _ in
                 List {
                     Section {
+                        NavigationLink(.settings(.currentAccount)) {
+                            HStack(spacing: 20) {
+                                AvatarView(
+                                    url: appState.profileTabRemoteSymbolUrl,
+                                    type: .user,
+                                    avatarSize: 60,
+                                    iconResolution: .unrestricted
+                                )
+                                    .padding(.vertical, -8)
```

Comment:
The spacing on this feels kinda off--what if we bump the avatar size down to 48 and ditch the negative padding?

Current:
<img width="368" alt="Screenshot 2023-12-07 at 12 12 28 PM" src="https://github.com/mlemgroup/mlem/assets/44140166/dd61b344-61d7-4f4a-a7d2-9ae20060f759">

Proposed:
<img width="363" alt="Screenshot 2023-12-07 at 12 11 51 PM" src="https://github.com/mlemgroup/mlem/assets/44140166/ef2ee4d3-b21d-4ac2-bc94-4f8512e7a7d3">

Reply:
Definitely agree that keeping things center-aligned is better. I still think the spacing is super tight--personally I think Apple dropped the ball on that one, but I'm content to leave as-is in accordance with our "follow Apple's lead" principle

Reply:
I don't mind reducing the current account icon size a little - the padding certainly is a little tight at the edges. I'd rather keep the alignment as is, though

Reply:
How's this?
![IMG_9724](https://github.com/mlemgroup/mlem/assets/78750526/07c81df5-fc44-4de9-b54d-0d4dfa16cf27)